src/psyclone/tests/psyir/symbols/typesymbol_test.py

```diff
@@ -58,3 +59,15 @@ def test_create_typesymbol_wrong_datatype():
         sym.datatype = "integer"
     assert ("datatype of a TypeSymbol must be specified using a "
             "DataType but got: 'str'" in str(err.value))
+
+
+def test_typesymbol_copy():
+    ''' Check that a TypeSymbol can be copied. '''
+    symbol = TypeSymbol("my_type", DeferredType(),
+                        visibility=Symbol.Visibility.PRIVATE,
+                        interface=UnresolvedInterface())
+    new_symbol = symbol.copy()
```

Comment:
Worth an `assert new_symbol is not symbol` too I think.

Reply:
Done